source/configuration_manual/authentication/user_database_extra_fields.rst

```diff
@@ -23,7 +23,6 @@ fields'. Possibilities are:
 * ``uidgid_file``: Get uid and gid for user based on the given filename.
 * ``user``: User can be overridden (normally set in passdb, see
   :ref:`authentication-password_databases`).
-* ``noreplicate``: See :ref:`replication_configuration`.
```

Comment:
This needs to be in the previous commit, otherwise that would yield errors and not produce any viable output. (Same as the other comment.)